### PR TITLE
Autocomplete keyframe animations when occluded on Windows 11

### PR DESCRIPTION
**Pull request by DHowett**
On certain builds of Windows, when Terminal is set as the default it
will accumulate an unbounded amount of queued animations while the
screen is off and it is servicing window management for console
applications.

This results in Terminal hanging when left overnight, as it has millions
of animations to process.

The new call into TerminalThemeHelpers will tell our compositor to
automatically complete animations that are scheduled while the screen is
off.

Fixes MSFT-38506980

### CODE REVIEW

**Comment by DHowett** [comment]
For servicing, this actually needs custom conflict resolution since this PR is built on the Common Nuget infrastructure.

**Review by lhecker — APPROVED** [review]

**Review by miniksa — APPROVED** [review]

File: src/cascadia/WindowsTerminal/AppHost.cpp [hunk 1]
```diff
@@ -399,6 +401,15 @@ void AppHost::Initialize()
     _revokers.OpenSystemMenu = _logic.OpenSystemMenu(winrt::auto_revoke, { this, &AppHost::_OpenSystemMenu });
     _revokers.QuitRequested = _logic.QuitRequested(winrt::auto_revoke, { this, &AppHost::_RequestQuitAll });
 
+    // BODGY
```
Comment (lhecker) [inline src/cascadia/WindowsTerminal/AppHost.cpp:404]: Do we have a MSFT:xxx ticket to track this with?

Comment (DHowett) [reply src/cascadia/WindowsTerminal/AppHost.cpp:404]: Nope. I'm following the linked ^^ bug from the PR body

**Comment by DHowett** [comment]
@msftbot merge this in 3 minutes

**Pull request merged**